oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -64,7 +64,12 @@ public Response execute(Client client, Identity identity, Set<String> scopes, Re
               .setSubject(identity.id())
               .setIssuedAt(instant.asDate())
               .setExpiration(new Date(accessToken.expirationTimestamp()))
-              .claim("identity", identityAsJson(identity).toString())
+              .claim("id", identity.id())
+              .claim("name", identity.name())
+              .claim("email", identity.email())
+              .claim("given_name", identity.givenName())
+              .claim("family_name", identity.familyName())
+              .claim("_private_claims", privateClaims(identity).toString())
```

Comment:
They could be hacked by the applications. Just use the same name that was stored as key in the map.